llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -596,6 +596,20 @@ class TargetLoweringBase {
   /// avoided.
   bool isJumpExpensive() const { return JumpIsExpensive; }
 
+  struct CondMergingParams {
+    int BaseCost;
+    int LikelyBias;
+    int UnlikelyBias;
+  };
+  // Return params for deciding if we should keep two branch conditions merged
+  // or split them into two seperate branches.
```

Comment:
Add paramater descriptions to the comment

Reply:
What do you mean by this?

Reply:
Typo "separate"

Reply:
Describe what the 3 function parameters do